generate-xcframework.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+while [[ $# -gt 0 ]]
```

Comment:
proponuję dodać po bin/bash w nowej linii
`set -e`

dzięki temu skrypt przerwie działanie jeżeli jedna komenda sfailuje,
nie będzie wywoływało lipo na build/xcframeworks jeżeli wcześniej mkdir się nie powiedzie itd.

aktualnie w przypadku faila każda kolejna komenda się wykona i gdyby coś poszło nie tak to będzie ciężko to zidentyfikować